shared/message/output/Buzzer.proto

```diff
@@ -0,0 +1,29 @@
+/*
+* This file is part of the NUbots Codebase.
+*
+* The NUbots Codebase is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* The NUbots Codebase is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+*
+* Copyright 2023 NUbots <nubots@nubots.net>
+*/
+
+syntax = "proto3";
+
+package message.output;
+
+
+/// @brief Buzzer message for the OpenCR subcontroller
```

Comment:
It would be good to consider this across subcontrollers. NUSense should have one too, so this buzzer message should be subcontroller-independent. This comment also doesn't really say much otther than its to do with the subcontroller. We already know its a buzzer message. Something like 'instructs to enable the buzzer on the subcontroller at the specified audio frequency'.